### DISCUSSION TITLE
Dark Mode: Project Boards

### DISCUSSION THREAD

**Discussion by oakeyc** [community/community#2912] [Projects and Issues] [answered]
On a project board with columns that have vertical scroll bars, the scroll bar is white. I think it would make more sense for the scroll bar to be a more grey or "dark" variant.

<img width="906" alt="Screen Shot 2020-12-09 at 3 08 05 PM" src="https://user-images.githubusercontent.com/16140724/101681666-7fafc900-3a30-11eb-8569-0fcf57bdcb96.png">

**Comment by becca** [comment, marked as answer]
Hello there and thank you for this feedback. Right now this feedback area is only for Discussions and GitHub for Mobile (https://github.com/github/feedback/discussions/1). 

All dark mode related issues can be shared in https://github.community/c/github-help/dark-mode-beta/65
